spec/classroom_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../classroom'
+require_relative '../student'
+
+describe Classroom do
```

Comment:
- [ Required ] I think it would be better if you add tests for the `add_student` method in the `Classroom` class, according to the project requirements you are expected to test all methods in the classes you created including the `Classroom` class. Please see the screenshot below and rectify this.

## Project requirements
![Screenshot from 2023-06-09 00-19-48](https://github.com/LoviMadaan/oop-school-library/assets/16392046/e50b3362-7540-4d9b-a450-83a1b4c934ba)